Tools-Override/tests.targets

```diff
@@ -27,7 +26,7 @@
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(RunningOnNetFx)' != 'true'">
+  <PropertyGroup Condition="'$(BuildingNETFxVertical)' != 'true'">
```

Comment:
Is using BuildingNETFxVertical the correct thing to do here? I see it is also set when we BuildAllConfigurations (https://github.com/dotnet/corefx/blob/e663503c8f7a40d9e54faff126732ebfda76de47/dir.props#L173) which would potentially lead to the fact that we have multiple test run configurations conflicting with each other.

cc @tarekgh @ericstj